From: Rick Jones <rick.jones2@hp.com>
Date: Mon, 21 Nov 2011 10:54:05 +0000
Subject: [PATCH 25/49] Sweep away N/A fw_version dustbunnies from the
 .get_drvinfo routine of a number of drivers
Patch-mainline: v3.3-rc1
Git-commit: 84b405011166e663fe9ef56c29b1d76f59b35568 (partial)
References: bsc#909566 FATE#314306

Per discussion with Ben Hutchings and David Miller, go through and
remove assignments of "N/A" to fw_version in various drivers'
.get_drvinfo routines.  While there clean-up some use of bare
constants and such.

Signed-off-by: Rick Jones <rick.jones2@hp.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Gary Ching-Pang Lin <glin@suse.com>
---
 drivers/net/ethernet/marvell/sky2.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/net/ethernet/marvell/sky2.c
+++ b/drivers/net/ethernet/marvell/sky2.c
@@ -3667,7 +3667,6 @@ static void sky2_get_drvinfo(struct net_
 
 	strlcpy(info->driver, DRV_NAME, sizeof(info->driver));
 	strlcpy(info->version, DRV_VERSION, sizeof(info->version));
-	strlcpy(info->fw_version, "N/A", sizeof(info->fw_version));
 	strlcpy(info->bus_info, pci_name(sky2->hw->pdev),
 		sizeof(info->bus_info));
 }
